_posts/2022-06-01-hhvm-4.161.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: "HHVM 4.161"
+layout: post
+author: alexeyt
+category: blog
+---
+
+HHVM 4.161 is released! 4.157 - 4.159 remain supported,
+as do the 4.128 and 4.153 LTS releases.
```

Comment:
Could you mention if there will be a release next week?

Reply:
There will be, but only because this one got delayed and should have happened last week. I didn't feel we needed to call that out.